src/components/Atoms/Button/Button.js

```diff
@@ -34,20 +41,31 @@ const StyledButton = styled.button`
   }
 `;
 
-const Button = React.forwardRef(({ children, wrapper, ...rest }, ref) => (
-  <StyledButton {...rest} as={wrapper ? 'span' : 'button'} ref={ref}>
-    {children}
-  </StyledButton>
-));
+const Button = React.forwardRef(
+  ({
+    children, backgroundColor = 'red', wrapper, ...rest
```

Comment:
Do we really need the default 'red' backgroundColor here if we're setting it via defaultProps?